packages/ethereum/node-tests/ethereum-indexer-test.js

```diff
@@ -0,0 +1,205 @@
+/* eslint-env node */
+
+const tokenAbi = require("./contracts/simple-erc20-abi");
+const {
+  createDefaultEnvironment,
+  destroyDefaultEnvironment
+} = require('@cardstack/test-support/env');
+const JSONAPIFactory = require('@cardstack/test-support/jsonapi-factory');
+
+describe('ethereum-indexer', function() {
+  let env, dataSource;
+
+  async function setup() {
+    let factory = new JSONAPIFactory();
+
+    dataSource = factory.addResource('data-sources')
+      .withAttributes({
+        'source-type': '@cardstack/ethereum',
+        params: {
+          branches: {
+            master: {
+              host: 'localhost',
+              port: 8545,
+              network_id: "*",
+            }
+          },
+          contracts: {
+            "sample-token": {
+              abi: tokenAbi,
+              location: {
+                network: "testrpc",
+                address: ""
+              },
+            }
+          }
+        },
+      });
+
+    env = await createDefaultEnvironment(`${__dirname}/..`, factory.getModels());
+
+    await env.lookup('hub:indexers').update({ realTime: true });
```

Comment:
createDefaultEnvironment does this for you automatically, it's not needed here.

Reply:
Cool thanks